_posts/2016-10-03-sr2017-kickstart-dates-announced.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Via http://www.hongkiat.com/blog/jekyll-draft/ it turns out that you can control the rendering of future items via `future: true` in your config file. Unfortunately Jekyll appears to need restarting to notice a change in the config file for me, but this is still probably nicer than changing the file name.

`_drafts` is the other option, though we'd then need to move the actual file in the repo at the end of the review process.